turbinia/workers/be.py

```diff
@@ -102,8 +102,7 @@ def run(src_path, num_workers, page_size=16777216):
 class BulkExtractorReducerTask(TurbiniaTask):
   """Reduce bulk extractor outputs."""
 
-  @staticmethod
-  def run(results):
+  def run(self, evidence, results):
     """Celery task that reduces the results into one SQLite database.
```

Comment:
Yep.  Removed.  There's still lots of other broken things to fix in this Task, but I'll tackle them later.